### PR TITLE
Add get API support for policy

### PR DESCRIPTION
**Pull request by zimengyang**
* add get API handler for policy
* TODO: add tests

### CODE REVIEW

File: pkg/identity-manager/handlers.go [hunk 1]
```diff
@@ -253,6 +254,26 @@ func (h *Handlers) getPolicies(params policyOperations.GetPoliciesParams, princi
...
+	defer trace.Tracef("get policy name '%s'", params.PolicyName)()
+	var policy Policy
+
+	name := params.PolicyName
+	if err := h.store.Get(IdentityManagerFlags.OrgID, name, entitystore.Options{}, &policy); err != nil {
+		log.Errorf("store error when getting policy '%s': %+v", name, err)
+		return policyOperations.NewGetPolicyNotFound().WithPayload(
+			&models.Error{
+				Code:    http.StatusNotFound,
+				Message: swag.String("policy not found when getting policy"),
```
Comment (neosab) [inline pkg/identity-manager/handlers.go]: Nit: `policy not found` would suffice

File: pkg/identity-manager/handlers.go [hunk 2]
```diff
@@ -253,6 +254,26 @@ func (h *Handlers) getPolicies(params policyOperations.GetPoliciesParams, princi
 	return policyOperations.NewGetPoliciesOK().WithPayload(policyModels)
 }
 
+func (h *Handlers) getPolicy(params policyOperations.GetPolicyParams, principal interface{}) middleware.Responder {
+
+	defer trace.Tracef("get policy name '%s'", params.PolicyName)()
+	var policy Policy
+
+	name := params.PolicyName
+	if err := h.store.Get(IdentityManagerFlags.OrgID, name, entitystore.Options{}, &policy); err != nil {
```
Comment (neosab) [inline pkg/identity-manager/handlers.go]: Can you add FilterExists() to the entititystore.Options.Filter, even though we hard delete policies I would like to future proof. Also, can you make a similar change to deletePolicy?

File: pkg/identity-manager/handlers.go (hunk 1 above)
Comment (zimengyang) [reply pkg/identity-manager/handlers.go]: fixed

File: pkg/identity-manager/handlers.go (hunk 2 above)
Comment (zimengyang) [reply pkg/identity-manager/handlers.go]: fixed

**Pull request merged**